Makefile

```diff
@@ -5,7 +5,7 @@ update:
 	./composer.phar update
 
 test: vendor/bin/phpunit build
-	./vendor/bin/phpunit --strict
+	./vendor/bin/phpunit
```

Comment:
Is there  a reason for this? Using less strict settings is usually bad practice.